doc/site/data/unsynced_commands.json

```diff
@@ -62,7 +62,7 @@
    "atm" : {
       "arguments" : {},
       "cheatRequired" : false,
-      "command" : "ATM",
+      "command" : "Atm",
```

Comment:
why?

Reply:
These files are autogenerated, they should be reverted as per my sibling comment, see also https://github.com/beyond-all-reason/RecoilEngine/issues/2457